ios/FluentUI/Label/Label.swift

```diff
@@ -137,6 +146,7 @@ open class Label: UILabel, TokenizedControlInternal {
     }
 
     private var _textColor: UIColor?
+    private var _font: UIFont?
```

Comment:
Is the issue that we weren't setting the tokens correctly where it's being used by other controls? Adding a new private var seems odd.

Reply:
Yeah, so if other controls wanted to override the font token, they would need to pass in the style in the Label init, but that would require a lot of changes in both our own controls and consumers directly using Label. Instead, we follow what text color is doing and override the font property of the UILabel.

Reply:
this would affect all attributedstring usuage as well? right?

Reply:
@sophialee0416  this would be problematic to your tableviewcell attributedstring largertext flow right?

Reply:
Attributed strings were still able to override the font as expected:
![IMG_0105](https://user-images.githubusercontent.com/31874971/223542963-8173ee55-7589-4ffb-bc81-be1bb33c8ebf.PNG)

Reply:
Yea so I can see that we're following the same pattern for font as we are for the textColor in this Label component here, which to me looks like it should be a safe fix. I remember we weren't super happy with the way textColor/font is handled here in general but not sure if that problem should be addressed in this PR